FluentAssertions.Net45.Specs/AsyncFunctionExceptionAssertionSpecs.cs

```diff
@@ -183,6 +183,29 @@ public void When_async_method_throws_exception_expected_not_to_be_thrown_it_shou
             action.ShouldThrow<AssertFailedException>()
                 .WithMessage("Did not expect System.ArgumentException, but found one*");
         }
+
+        [TestMethod]
+        public void When_async_method_throws_exception_without_inner_exception_it_should_throw_AssertFailedException()

```

Comment:
I don't get the rationale. Do you mean that it should not cause an internal error? 

PS. See also point 11 from [this post](http://www.continuousimprover.com/2015/11/12-tips-to-write-unit-tests-that-dont.html).